app/assets/stylesheets/mobile.sass.scss

```diff
@@ -102,7 +102,25 @@ $displayfont: "OPTIOliver-Display";
   border-bottom: 1px solid $primary;
 
   .navbar-brand {
+    font-family: "OPTIOliver-Display";
+    text-transform: uppercase;
     text-decoration: none;
+
+    span {
+      margin-right: 0.1em;
+    }
+
+    span:nth-of-type(1) {
+      color: var(--bs-navbar-brand-hover-color);
+    }
+
+    span:nth-of-type(2) {
+      color: $tertiary;
+    }
+
+    span:nth-of-type(3) {
+      color: $primary;
+    }
```

Comment:
Is there a motivation behind using `nth-of-type` rather than a class to make it more declarative in the stylesheet? (Project doesn't seem very BEM-y, but could still use something to describe what the first, second and third `span`s contain?)

Reply:
I personally think we often get hung up on that sort of approach. The classes would have no more meaning than `first`, `second`, `third`, so I don't see much value in adding them. It's not like there's some hidden semantics in here, it is really just "colour each word differently".